src/main/java/simpli/tasks/Task.java

```diff
@@ -0,0 +1,59 @@
+package simpli.tasks;
+
+public class Task {
+    private final String name;
+    private boolean isDone;
+
+    /**
+     * Initializes a task with the specified attributes.
+     * @param isDone boolean representing if a task is completed or not.
```

Comment:
Could we add an empty line between the description and parameter section here similar to the other method headers?